### PR TITLE
Feat/spotlight

### PR DESCRIPTION
**Pull request by FabianRuettgers**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ const nextConfig = {
   compiler: {
     styledComponents: true,
   },
+  env: { API_KEY: "4c606aec01f0a1c7daed75110995b085" },
```
Comment (jonas8900) [inline next.config.js:6]: What does this mean?

Comment (FabianRuettgers) [reply next.config.js:6]: The API I use needs an environment variable. I tried it with an env.local file, but this doesn't work, so I looked it up and found this method.

File: pages/_app.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 import GlobalStyle from "../styles";
 
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
 export default function App({ Component, pageProps }) {
   return (
     <>
-      <GlobalStyle />
-      <Component {...pageProps} />
+      <SWRConfig value={{ fetcher, refreshInterval: 1000 }}>
```
Comment (jonas8900) [inline pages/_app.js]: Why do you use a refreshInterval here?

**Review by jonas8900 — COMMENTED** [review]
the rest looks good to Miejke and me

File: pages/_app.js (hunk 2 above)
Comment (FabianRuettgers) [reply pages/_app.js]: I use it to make sure that the trending page is always updated, as it is possible that the data in the fetched array changes.

Comment (FabianRuettgers) [reply pages/_app.js]: Sorry, I just recheck, this doesn't appear to be working as I thought. I deleted it.

**Pull request merged**